packages/cli/src/commands/promote/request-promote.ts

```diff
@@ -70,11 +70,20 @@ export default async function requestPromote({
     );
     return 0;
   }
-  await client.fetch(`/v10/projects/${project.id}/promote/${deployment.id}`, {
-    body: {}, // required
-    json: false,
-    method: 'POST',
-  });
+  const res = await client.fetch(
+    `/v10/projects/${project.id}/promote/${deployment.id}`,
+    {
+      body: {}, // required
+      json: false,
+      method: 'POST',
+    }
+  );
+  if (res.status === 202) {
+    output.log(
+      'Promotion has been queued and will begin when the active rolling release completes successfully.'
```

Comment:
In other commands that take a while, we some times long poll so we can show the result to the user with a `--no-wait` option for CI uses where waiting is not required. Not strictly necessary here if that doesn't match how this feature works, so I wouldn't block on it but maybe consider adding.

I _could_ see this being useful for custom workflows where a failed promote exits with a non `0` and their script needs to take some kind of action.

Alternatively, is there a URL we can log immediately for a user to visit so they can watch the rolling release progress?